mash_client/cli_utils.py

```diff
@@ -99,10 +99,14 @@ def get_config(cli_context):
     }
     data = ChainMap(cli_values, config_values, defaults)
 
+    host = data['host']
+    if not host.startswith('http'):
+        host = ''.join(['http://', host])
```

Comment:
Should this be https and not http?

Reply:
> Should this be https and not http?

Well, it depends on server side setup.

1.) Server only listens on 443 (https) - our setup
2.) Sever listens on 80 (http) and 443 and answers to both
3.) Server listens on 80 and 443 but only answers to https, i.e. 80 is redirected to 443

I think this is a documentation item rather than a code item. But if implemented in code it should have a fall through, try http, if it fails switch to https and possibly update the config automatically.

Reply:
The goal here was only to provide a default protocol if the user does not supply it in config of cmd line. The Requests package requires there to be a protocol but will not add one. We can only take one guess and I think http should be default.

We shouldn't try to solve the url for the user, but I could add info in help message to make it clear that http or https is required.

My opinion is we either have a simple default of http if missing or we drop this PR because trying to solve the URL is overkill especially if the help message is clear about requiring the protocol.

Reply:
I am fine just documenting the option more clearly